### ISSUE TITLE
Time left on print based on layer completion time is no longer functioning.

### ISSUE THREAD

**Issue by MrSteve920** [Duet3D/PanelDueFirmware#131]
Per the Duet RepRap firmware 3.3 changelog: "Estimated print time provided by the slicer (using M73 commands if present) is now available. Estimated print completion time based on layer progress has been removed."

Because of this change, the paneldue is displaying N/A for the time left on a print based on layer completion on the Print menu.

This should be changed to display the time left on a print based on the slicer reported time remaining instead of the layer completion time. Values reported by slicer via M73 command should be displayed if available, otherwise paneldue should be displaying "slicer N/A" instead of "layer N/A".

Time left on a print based on the slicer reported time via M73 is being properly displayed in Duet Web Control.

- Paneldue hardware v2.
- Paneldue firmware 3.2.11. Issue was also present on 3.2.9.
- Reprap firmware 3.3 stable branch

![Screenshot 2021-06-18 074707](https://user-images.githubusercontent.com/41220784/122556373-6acc7780-d009-11eb-8f33-af4f4d4012a3.png)

**Comment by mfs12** [comment]
@MrSteve920 can you please provide the following information:

- which paneldue hardware are you using?
- which paneldue firmware version are you running?
- which reprap firmware are you running?

**Comment by MrSteve920** [comment]
- Paneldue hardware v2. 
- Paneldue firmware 3.2.11. Issue was also present on 3.2.9.
- Reprap firmware 3.3 stable branch

**Comment by dc42** [comment]
RRF no longer generates print completion time estimated based on layer completion. See the RRF 3.3 upgrade notes.

**Comment by mfs12** [comment]
https://forum.duet3d.com/topic/21735/poll-time-estimates-on-paneldue-5-7/23

**Comment by MrSteve920** [comment]
I think we're all in agreement that reprap 3.3 removed the print time estimate based on layer completion. I already indicated that I saw this was removed per the 3.3 upgrade notes.

I'm suggesting that moving forward the paneldue should be updated to no longer display the time left based on layer on reprap 3.3 onward the display will continue to show "layer n/a" which is fairly useless now that this has been removed.

This could be resolved by doing the following per my original comment:
This should be changed to display the time left on a print based on the slicer reported time remaining instead of the layer completion time. Values reported by slicer via M73 command should be displayed if available, otherwise paneldue should be displaying "slicer N/A" instead of "layer N/A".

An alternative option:
If it's not possible to display the time estimated based on slicer provided values then it would at least be cleaner to have the paneldue not display "layer N/A" if reprap firmware >=3.3 is being used.
